### PR TITLE
Adding Okta Access documentation to Readme

### PR DESCRIPTION
**Pull request by jroblesx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,9 +153,31 @@ Create a PR with changes and request review. Once approved, the branch will be
...
+2. Determine the new semantic version number
+3. Create the version and tag with `yarn version [--major] [--minor] [--patch]`
+4. Push the commit and tag with `git push --follow-tags`
 
-That's it! Travis will deploy the necessary bits to NPM. Manual deployment is
+That's it! [Travis](https://www.travis-ci.com) will deploy the necessary bits to NPM. Manual deployment is
 possible of course, just be certain to follow the `yarn build` road.
+
+### How to create a new Okta API token and configure environment variables
+1. Require **Super Admin** access to the [development account](https://dev-857255.okta.com/).
```
Comment (Gonzalo-Avalos-Ribas) [inline README.md:164]: Would love that the instructions also cover how to grant *Super Admin* to the user we are adding.
We should set it up to make it possible for the added user to add another user if needed.

File: README.md [hunk 2]
```diff
@@ -153,9 +153,31 @@ Create a PR with changes and request review. Once approved, the branch will be
...
+2. Login into the *Okta Admin Dashboard* using [this link](https://dev-857255-admin.okta.com/) and your credentials.
+3. Navigate to *Security > API > Tokens*.
+
+![navigateToTokens](/docs/images/navigateToTokens.gif)
+
+4. Now you need to create a new token following the next steps:
```
Comment (Gonzalo-Avalos-Ribas) [inline README.md:170]: Create a new token following the next steps:

**Comment by Gonzalo-Avalos-Ribas** [comment]
This is **awesome**! Thanks!

File: README.md (hunk 2 above)
Comment (jroblesx) [reply README.md]: Text changed!

File: README.md [hunk 3]
```diff
@@ -153,9 +153,50 @@ Create a PR with changes and request review. Once approved, the branch will be
...
+### How to invite a new User to the Okta development account
+
+1. Access to Okta [Admin Dashboard](https://dev-857255-admin.okta.com/)
+2. Navigate to _Directory > People_.
+   ![navigateToPeople](/docs/images/navigateToPeople.gif)
+
+3. Click on _Add Person_ button.
+   ![openAddPersonForm](/docs/images/clickAddPerson.gif)
+
+4. Fill all the required fields, select **J1_Devs** in the _Groups_ input and
```
Comment (Gonzalo-Avalos-Ribas) [inline README.md:198]: User type seems to be left as user, is it possible to grant admin?

File: README.md [hunk 4]
```diff
@@ -153,9 +153,50 @@ Create a PR with changes and request review. Once approved, the branch will be
...
+3. Create the version and tag with `yarn version [--major] [--minor] [--patch]`
+4. Push the commit and tag with `git push --follow-tags`
 
-That's it! Travis will deploy the necessary bits to NPM. Manual deployment is
-possible of course, just be certain to follow the `yarn build` road.
+That's it! [Travis](https://www.travis-ci.com) will deploy the necessary bits to
+NPM. Manual deployment is possible of course, just be certain to follow the
+`yarn build` road.
+
+### How to create a new Okta API token and configure environment variables
```
Comment (Gonzalo-Avalos-Ribas) [inline README.md:164]: This is great, i would leave it after the ### How to invite a new User to the Okta development account

File: README.md [hunk 5]
```diff
@@ -153,9 +153,50 @@ Create a PR with changes and request review. Once approved, the branch will be
...
+2. Navigate to _Directory > People_.
+   ![navigateToPeople](/docs/images/navigateToPeople.gif)
+
+3. Click on _Add Person_ button.
+   ![openAddPersonForm](/docs/images/clickAddPerson.gif)
+
+4. Fill all the required fields, select **J1_Devs** in the _Groups_ input and
+   press the _Save_ button. ![fillNewPersonForm](/docs/images/fillUserForm.gif)
+
+After pressing the _Save_ button the new user must receive an email with
```
Comment (Gonzalo-Avalos-Ribas) [inline README.md:201]: After executing all this steps, is it possible for the new user to control all the okta resources? Should admin be granted in any other way?

**Review by Gonzalo-Avalos-Ribas — COMMENTED** [review]
This is great, left a few comments. The most important thing is that this docuentation actually belongs on the integration-deployments repo, and not here. So we should move it.

File: README.md (hunk 3 above)
Comment (jroblesx) [reply README.md:198]: The group has **Super Administrator** permission in behind, so when we select this group we are also adding our new user as **Super Administrator**.

File: README.md (hunk 5 above)
Comment (jroblesx) [reply README.md:201]: Yes, the only one role that has the nested permissions to perform all the api operations we use is **Super Administrator**, that means that new users will be able to perform administrative tasks, this is just for our internal dev environment, we are working in a separated task (user facing) on more granular permissions and a different procedure.

**Pull request closed without merging**